graph_creator/models.py

```diff
@@ -105,7 +108,7 @@ def worlds_students_per_country_statistics(cls):
             world_students_per_country (dict): Country-count accordance as pair of key-value.
         """
 
-        start_of_day, end_of_day = get_previous_day_start_and_end_dates()
+        start_of_day, end_of_day = get_previous_day_start_and_end_dates(backtrack=1)
```

Comment:
To make it work for fixtures, put backtrack=20, if there is data for 20 days ago.